### ISSUE TITLE
"Server" reporting to New Relic is lost on reboot.

### ISSUE THREAD

**Issue by larspetrus** [thewca/worldcubeassociation.org#1448]
See: https://github.com/phlipper/chef-newrelic-sysmond/issues/26

Currently, you have to run chef to make this work, and it's lost when rebooting the machine.

Could we do whatever chef does during reboot?

**Comment by gregorbg** [comment]
Is this issue still relevant?

**Comment by FinnIckler** [comment]
I don't think we even use new-relic anymore (at least I can't remember anyone having looked at it since I am here). I vaguely remember a discussion 2 years ago about how the advanced monitoring feature for RDS was better anyway for the metrics.

**Comment by viroulep** [comment]
RDS works just for the db, new relic is rails oriented: you get detailed profiling data about the various application layers; it's definitely excellent information!
I can't remember if it still works, but I'd definitely advice to keep it if possible, even though we haven't had performance issue for some time now, due to the low amount of competitions.

As for this specific issue I actually don't know if it's still relevant, but it should be easy to check.

**Comment by FinnIckler** [comment]
Ah, didn't know we had anything set up for application metrics already. Would be good to look at, as we are getting alerts for the site being unavailable every couple of days, but we haven't figured out why and never caught it happening in real time.

**Comment by viroulep** [comment]
I think data are kept for quite sometime, so you may actually be able to log in new relic and take a look at the time frame where it happened!

**Comment by dunkOnIT** [comment]
No longer relevant
